services/MerchStoreService.ts

```diff
@@ -44,7 +46,7 @@ export default class MerchStoreService {
     const collection = await this.transactions.readOnly(async (txn) => Repositories
       .merchStoreCollection(txn)
       .findByUuid(uuid));
-    if (!collection) throw new NotFoundError('Collection not found');
+    if (!collection) throw new NotFoundError(NotFoundErrors.MERCH_COLLECTION);
```

Comment:
```suggestion
    if (!collection) throw NotFoundErrors.MERCH_COLLECTION;
```

Reply:
Doesn't this not throw a 404 error then and instead throw a 500 (the default error code defined in `errors/handler.ts`)?

Reply:
The idea's that `NotFoundErrors.MERCH_COLLECTION` would be `new NotFoundException(message)`. Nicer caller syntax.

Reply:
I'll keep this in mind but will hold off till a separate, error refactor PR.